### ISSUE TITLE
Footer: remove existing email section, update footer, and add new 'email' signup to footer

### ISSUE THREAD

**Issue by natalieworth** [mozilla/foundation.mozilla.org#2834]
We'll be removing the existing email signup section on the Foundation Site and moving it to the footer area. Desktop and mobile versions of the regular, error, and success states are in the [Redpen here.](https://redpen.io/p/bp40e77cb408ac932a)

Removing this section:
<img width="1569" alt="Screen Shot 2019-03-17 at 11 10 50 PM" src="https://user-images.githubusercontent.com/14876567/54510292-f2e2e480-4909-11e9-96e3-0fe7c18731db.png">

Desktop:
<img width="976" alt="Screen Shot 2019-03-17 at 11 07 36 PM" src="https://user-images.githubusercontent.com/14876567/54510179-8e278a00-4909-11e9-8a80-27a8278f711a.png">

Mobile:
<img width="249" alt="Screen Shot 2019-03-17 at 11 07 50 PM" src="https://user-images.githubusercontent.com/14876567/54510185-92ec3e00-4909-11e9-9f95-08acfda76450.png">

**Comment by natalieworth** [comment]
Icons (check, twitter, instagram, and github) are here:
[icons.zip](https://github.com/mozilla/foundation.mozilla.org/files/2976560/icons.zip)

**Comment by mmmavis** [comment]
@natalieworth 2 questions for ya:

1. Which Sketch file can I find this updated footer design?
2. Opportunity pages can include a sign up form (see [test page](https://foundation.mofostaging.net/en/opportunity/test-opportunity-page-with-sign-up-form/) on staging). Should we file a ticket to update the form design so it aligns better with the rest of the site?

**Comment by natalieworth** [comment]
After meetings with Sabrina yesterday to talk about brand updates, I made a few minor changes to the button styling for this and have updated the [design files here.](https://redpen.io/p/bp40e77cb408ac932a)

The last thing I need to confirm is the new dark (inverse) style with Sabrina tomorrow morning and I can annotate these in redpen/ github with the style change notes for devs. Thanks for waiting for the design changes! Lots of small tweaks! :)

**Comment by natalieworth** [comment]
@mmmavis Is the Opportunity page form different than the footer? I'm not sure how I understand how that page works. Is there something that exists on the site or is it only on staging.

**Comment by mmmavis** [comment]
@natalieworth it's part of what the Opportunity Page template offers so it's available on all production/staging/review apps. You can try creating a test page on review app or staging (or production, but make sure you don't publish it live) to see how things are tied together.

<img width="1085" alt="image" src="https://user-images.githubusercontent.com/2896608/54629732-659cae80-4a35-11e9-975f-ab69bc8c39e1.png">

Basically CMS editors can create different sign up forms. A Opportunity Page can then choose which sign up form it wants to include.

**Comment by mmmavis** [comment]
One example on production site: https://foundation.mozilla.org/en/opportunity/mozilla-information-trust-initiative/

**Comment by kristinashu** [comment]
As discussed in Slack... @alanmoo and I have talked about removing the the signup form snippet from the CMS so don't worry about styling that component. We want to drive people to our main mozilla email and restrict the number of bespoke email lists. The exception will be MozFest which will have it's own signup but will be styled like a campaign petition on the right side of the page.

**Comment by mmmavis** [comment]
@natalieworth can you mock error state for the form as well?

**Comment by natalieworth** [comment]
@mmmavis I forgot to add the error exclamation mark icon. I've uploaded all the svg icons in here:
[icons.zip](https://github.com/mozilla/foundation.mozilla.org/files/2990087/icons.zip)

**Comment by mmmavis** [comment]
@natalieworth what messages do we wanna show and where to show them when

1. sign-up form has successful gone through
2. a server error upon form submission (e.g., This rarely happens but there could be time that server goes down after user's passed form validation and clicked 'sign up')

**Comment by natalieworth** [comment]
Hey @mmmavis , so sign-up/ success is here https://redpen.io/vd414f4e668966a073
I didn't consider the server error. Do you know what example copy for that would look like? Or could it be something like: "Something went wrong" and "Internal Server Error"? (Is it the[ 500 error](https://images.drivereasy.com/wp-content/uploads/2017/10/Youtube-500-Internal-Server-Error.jpg)?)

**Comment by alanmoo** [comment]
> As discussed in Slack... @alanmoo and I have talked about removing the the signup form snippet from the CMS so don't worry about styling that component. We want to drive people to our main mozilla email and restrict the number of bespoke email lists. The exception will be MozFest which will have it's own signup but will be styled like a campaign petition on the right side of the page.

Just to +1 this, let's build the designs of this ticket into the templates at a pretty base layer.

As for error messages, we can make it say something generic ("Something went wrong"). Usually, that's in the same place as a message when someone's email is invalid; Do we have styling for that yet too?

**Comment by mmmavis** [comment]
@kristinashu @natalieworth any idea where we should link Instagram icon to?

**Comment by alanmoo** [comment]
I'm guessing we shouldn't, given it's Facebook owned?

**Comment by kristinashu** [comment]
I suggest following what mozilla.org does in their footer and link to https://www.instagram.com/mozilla/

**Comment by mmmavis** [comment]
@natalieworth 

https://redpen.io/qx2f1a26d415e0a1b1 Could we make the privacy checkbox line wrap at email input box's width?

We can't easily implement things in a way that is not hacky, look right design-wise, and in right tab order. TL;DR is that in order to make email input and submit button sit side by side & the privacy checkbox line full width, we have to render `[email input] [submit button]` first... which results in tab order goes from `email` -> `submit` -> `privacy checkbox`. Ideally we want the tab order to go from form fields first and submit button last.

**Comment by natalieworth** [comment]
@mmmavis Yes! That's totally fine if that works and is easiest for you. I don't think it will compromise the design.

**Comment by mmmavis** [comment]
@natalieworth awesome! 🎉

**Comment by kristinashu** [comment]
👏  I think this is good to close 👏  just a little follow up here #3007 👏
